### ISSUE TITLE
Update Park City, UT neighbourhoods

### ISSUE THREAD

**Issue by stepps00** [whosonfirst-data/whosonfirst-data#1264]
### tl;dr

Status:
- [x]  Propose sources
- [x]  Establish concordances between WOF and source
- [x]  Upgrade / add WOF shapes
- [x]  Add new or missing features to WOF
- [x]  Deprecate / supersede WOF features
- [x]  Draft review
- [x]  Draft revisions based on review comments
- [x]  Final review
- [x]  Final revisions based on review comments
- [x]  Import into Who's On First
### Details

Sourcing:
- Permissive license?
- Add new src to Who's On First

Upgrade / add:
- `macrohood`
- `neighbourhood`
- `microhood`

Add new or missing features to WOF
- `macrohood`
- `neighbourhood`
- `microhood`

Deprecate / supersede:
- `macrohood`
- `neighbourhood`
- `microhood`

**Comment by stepps00** [comment]
WOF does not have neighbourhood coverage in Park City, so this import should be simple (like Santa Barbara, #1005).

`CC0` source, based off of Park City, UT Planning Dept docs, then clipped with the existing WOF locality shape: https://gist.github.com/stepps00/b4ca67ec3b62ad31897e1fc1c2aa69bc.
